.github/workflows/ci.yml

```diff
@@ -5,26 +5,44 @@ on: [push, pull_request]
 
 jobs:
   test:
+    name: Run workflow tests
     runs-on: ubuntu-latest
     strategy:
       matrix:
         nxf_ver: ['19.10.0', '']
         aligner: ["--aligner 'hisat2'", "--aligner 'star'", "--pseudo_aligner 'salmon'"]
         options: ['--skipQC', '--remove_rRNA', '--saveUnaligned', '--skipTrimming', '--star_index false']
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out pipeline code
+        uses: actions/checkout@v2
+
+      - name: Check if Docker / Conda changed
+        uses: technote-space/get-diff-action@v1
+        with:
+          PREFIX_FILTER: |
+            Dockerfile
+            environment.yml
+
+      - name: Build new docker image
+        if: env.GIT_DIFF
+        run: docker build . -t nfcore/rnaseq:dev
```

Comment:
Just need to have a condition here for release (or have that in bump-version for the release)

Reply:
Yeah, I think that this will need to be changed in the `bump-version` code, as the code versions change in more CI events than just the release.

Reply:
Added to the TODO in the top comment 👍